src/MetaMorpho.sol

```diff
@@ -284,6 +285,18 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         }
     }
 
+    /// @notice Submits a forced market removal from the vault, losing all funds supplied to the market.
+    /// @dev To remove it from the withdraw queue after the timelock, the market also needs to have a zero cap.
+    /// @dev Warning: Submitting a forced removal will overwrite the timestamp at which the market will be removable.
+    function submitMarketRemoval(Id id) external onlyCuratorRole {
```

Comment:
```suggestion
    function submitMarketForceRemoval(Id id) external onlyCuratorRole {
```

Reply:
I think it's unnecessary and verbose, wdyt @morpho-org/onchain-team ?

I understand though that a "market removal" can be done in 2 ways:
1. via this timelocked function then with update withdraw queue
2. via cap set to zero, reallocation and update withdraw queue

And it may be confusing

Reply:
I think it's too verbose as well and I would keep the current version